package/boot/uboot-envtools/files/ramips

```diff
@@ -25,6 +25,10 @@ allnet,all0256n-8m|\
 allnet,all5002)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x10000" "0x10000"
 	;;
+ampedwireless,ally-00x19k|\
+ampedwireless,ally-r1900k)
+	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x20000" "4"
```

Comment:
What's the "4"?

Reply:
The uboot-env partition on these devices is massive (512KiB) relative to the actual environment size.  There are 4 0x20000 sectors in the partition.

Reply:
The u-boot-env on this device is huge:  512KiB.  Those options to ubootenv_add_uci_config() are the sector size & num sectors.